src/parser/shaman/enhancement/modules/Abilities.js

```diff
@@ -30,8 +30,9 @@ class Abilities extends CoreAbilities {
       },
       {
         spell: SPELLS.FERAL_SPIRIT,
+        buffSpellId: SPELLS.FERAL_SPIRIT.id, //This won't work if Elemental Spirits is taken since that gives a different buff
         category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-        cooldown: 120,
```

Comment:
Might be able to use ``buffSpellId: [SPELLS.FERAL_SPIRIT.id, SPELLS.ELEMENTAL_SPIRITS_BUFF.id],`` if not you can use a ternary like you do in cooldown?